cortex.yaml

```diff
@@ -11,7 +11,7 @@ info:
   x-cortex-tag: hivex
   x-cortex-type: service
   x-cortex-domain-parents:
-  - tag: vm-platform
+  - tag: endpoint-prism
```

Comment:
This should probably be assigned to the @rapid7/mavericks team.

Reply:
Ah never mind, this is used in the patcher service which the @rapid7/mavericks have handed over to the @prism team so this makes sense.